### ISSUE TITLE
Make the CXF Bus customizable (features, extensions, properties)

### ISSUE THREAD

**Issue by mickaeltr** [cloudsoft/winrm4j#81]
Hello,

First of all, many thanks for this library that is really useful to me.

I just hit a limit though. For debugging purposes, I wanted to log SOAP requests and responses. This is possible through a LoggingFeature applied on the CXF Bus. However, the WinRmClientContext bus field is not accessible. The trick for me was to use dirty reflection in order to customize it…

```java
WinRmClientContext context = WinRmClientContext.newInstance();

Bus bus;
try {
    bus = (Bus) FieldUtils.readField(context, "bus", true);
} catch (IllegalAccessException e) {
    throw new RuntimeException(e);
}

LoggingFeature loggingFeature = new LoggingFeature();
loggingFeature.setPrettyLogging(true);
bus.setFeatures(Collections.singletonList(loggingFeature));
```

It would be nice and easy to have that bus object more easily customizable.

Cheers

**Comment by aledsage** [comment]
This has been fixed in the 0.6.1 release (see https://github.com/cloudsoft/winrm4j/pull/91)

**Comment by mickaeltr** [comment]
Thanks @aledsage
